### PR TITLE
Advice

### PR DESCRIPTION
**Pull request by GMHanlon**

### CODE REVIEW

**Comment by redrambles** [comment]
Hey @GMHanlon !

We're close!! Right now, your advice file is inside a directory called `your-name-advice.html`, like this: `your-name-advice.html/gillian-hanlon-advice.html`

What we want is to move your `gillian-hanlon-advice.html` into the existing `advice` directory. Once you move it there, please delete the (now empty) `your-name-advice.html` directory.

Once these are complete, add your changes with `git add -A`, commit as per usual and push up your `advice` branch once more, which will automatically update this pull request. Then we should be all set to merge in your code!

Let us know if you have any questions!

Cheers,
Ann :)

**Comment by twosavoie** [comment]
Hi @GMHanlon,

Just checking to see if you need any help with this pull request. Reach out if you have any questions! 🙂 

Best,
Lisa

**Comment by GMHanlon** [comment]
Hi Lisa,

I used the ls command and this showed up on my command line: ``advice/  css/  index.html  your-name-advice.html/``

**Comment by twosavoie** [comment]
Hi @GMHanlon,

Well done running the `ls` command. You can see in the list, there's a directory named your-name-advice.html. Inside of that directory is your advice file. 

In Finder (Mac) or File Manager (Windows), open the your-name-advice.html directory. You can drag your advice file out and drop it on the desktop for now. Then delete the your-name-advice.html directory. 

Next, drag your advice file from the desktop into the advice directory. Once you've done that, you're good to go! Add your changes with git add -A, commit the changes, and push up your advice branch once more. Your pull request will be updated automatically. Then we should be all set to merge in your code!

Let us know if you have any questions!

Best,
Lisa

File: index.html [hunk 1]
```diff
@@ -16,7 +16,11 @@ <h2>Our Advice for Newbies!</h2>
     </header>
 
     <section>
+<<<<<<< HEAD
+      <a href="advice/gillian-hanlon-advice.html">Gillian's General Guidance</a><br>
+=======
```
Comment (redrambles) [inline index.html:21]: Remove this

File: index.html [hunk 2]
```diff
@@ -16,7 +16,11 @@ <h2>Our Advice for Newbies!</h2>
...
       <a href="advice/yoko-scott-advice.html"> target="_blank" Yoko Scott's Advice</a><br>
+>>>>>>> upstream/main
```
Comment (redrambles) [inline index.html:23]: Remove this

**Comment by redrambles** [comment]
Hey @GMHanlon !

Ok we are in great shape now, file structure wise! 🙌 

The only thing left to do is remove the conflict markers that remain in the index.html file. I left you 3 comments with the specific lines that need to be removed. Once you do that, add your change, commit, and push up your `advice` branch once more, and we should be all set!

Let us know if you have any questions!

Cheers,
Ann :)

File: index.html (hunk 1 above)
Comment (redrambles) [reply index.html:21]: Just the <<<<<HEAD and the ====== 

not your link! :)

Comment (GMHanlon) [reply index.html:21]: Hi Ann. I removed the info from my code editor. I don't know how its still showing up on Github.

Comment (redrambles) [reply index.html:21]: Ok so you'll need to save the file, add and commit the change in your Terminal, and then push up you branch again. @GMHanlon 

Then the changes should make it here.

**Comment by redrambles** [comment]
Hey @GMHanlon !

Yay!! Well done, your code is merged in! 🎉 

Cheers,
Ann :)

**Pull request merged**